sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -646,24 +647,23 @@ public Symbol resolveField(QualifiedName qualifiedName, @Nullable List path, Ope
             null
         );
 
-        Symbol symbol = expressionAnalyzer.convert(node.functionCall(), context.expressionAnalysisContext());
+        ExpressionAnalysisContext expressionContext = context.expressionAnalysisContext();
+        // we always require a Function symbol in this context
```

Comment:
```suggestion
        // we support `FROM scalar()` but not `FROM 'literal'` -> we turn off eager normalization so we can distinguish between Function and Literal. 
```

Reply:
I thought the code check just after was good enough as an explanation but this looks better I have to admit. It's just that in most cases we follow a 'do not comment on the obvious' pattern...

Thanks for the suggestion.